avivator/src/components/Controller/components/RenderingModeSelect.js

```diff
@@ -7,23 +7,26 @@ import InputLabel from '@material-ui/core/InputLabel';
 import { RENDERING_MODES } from '@hms-dbmi/viv';
 import { useImageSettingsStore, useViewerStore } from '../../../state';
 
+const renderingOptions = Object.values(RENDERING_MODES);
+
 function RenderingModeSelect() {
   const { setImageSetting, renderingMode } = useImageSettingsStore();
-  const { isViewerLoading } = useViewerStore();
+  const { isViewerLoading, use3d } = useViewerStore();
+  const options = !use3d ? [...renderingOptions, ''] : renderingOptions;
```

Comment:
what is the empty option?

Reply:
I like the look of it with the "Rendering Modes" label displaying when there is no selection (i.e when not in 3D mode).  Just like with the colormapping.

Reply:
Sounds good. Could you add a small comment describing that intention, otherwise its hard to know the effect of the `[..., '']` ?